web/pdf_page_view.js

```diff
@@ -534,26 +534,28 @@ var PDFPageView = (function PDFPageViewClosure() {
     beforePrint: function PDFPageView_beforePrint(printContainer) {
       var CustomStyle = pdfjsLib.CustomStyle;
       var pdfPage = this.pdfPage;
-
+      // increase viewport or DPI to improve quality
       var viewport = pdfPage.getViewport(1);
       // Use the same hack we use for high dpi displays for printing to get
       // better output until bug 811002 is fixed in FF.
-      var PRINT_OUTPUT_SCALE = 2;
+      var DPI = 144; 
+      var PRINT_OUTPUT_SCALE = DPI/72;
```

Comment:
Nit: To me, this seems kind of orthogonal to the current patch which tries to fix printing cut-off in IE, so I'd suggest just reverting this to `var PRINT_OUTPUT_SCALE = 2;` instead.